charts/kueue/templates/crd/kueue.x-k8s.io_topologies.yaml

```diff
@@ -94,26 +94,3 @@ spec:
         type: object
     served: true
     storage: true
-  - name: v1beta1
-    schema:
-      openAPIV3Schema:
-        properties:
-          levels:
-            description: levels define the levels of topology.
-            items:
-              type: string
-            maxItems: 8
```

Comment:
Does `kubectl get topologies.kueue.x-k8s.io` work if you set the conversion strategy to None?